net-p2p/bitcoind/bitcoind-0.16.0.ebuild

```diff
@@ -0,0 +1,156 @@
+# Copyright 2010-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DB_VER="4.8"
+inherit autotools bash-completion-r1 db-use systemd user
+
+BITCOINCORE_COMMITHASH="4b4d7eb255ca8f9a94b92479e6061d129c91a991"
+KNOTS_PV="${PV}.knots20180322"
+KNOTS_P="bitcoin-${KNOTS_PV}"
+
+DESCRIPTION="Original Bitcoin crypto-currency wallet for automated services"
+HOMEPAGE="https://bitcoincore.org/ https://bitcoinknots.org/"
+SRC_URI="
+	https://github.com/bitcoin/bitcoin/archive/${BITCOINCORE_COMMITHASH}.tar.gz -> bitcoin-v${PV}.tar.gz
+	https://bitcoinknots.org/files/0.16.x/${KNOTS_PV}/${KNOTS_P}.patches.txz -> ${KNOTS_P}.patches.tar.xz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux ~arm ~arm64 ~mips ~ppc ~x86 ~x86-linux"
+IUSE="+asm +bitcoin_policy_rbf examples knots libressl test upnp +wallet zeromq"
+
+DEPEND="
```

Comment:
Could you sort the dependencies by cat/pn, with unconditionals first and conditionals (sorted by USE flag) later? It would make it easier to distinguish those complex deps.